### DISCUSSION TITLE
Facial Recognition Icon

### DISCUSSION THREAD

**Discussion by ghost** [organicmaps/organicmaps#1039] [Ideas]
I hear this wonderful team is planning to add an icon of person with bag over face to indicate places using facial recognition. That sounds like a great idea that will make many people want this map to guide them safely through the world.

**Comment by biodranik** [comment]
Where did you hear that? :)

Is there already such type in OpenStreetMap data?

**Reply by rtsisyk** [reply]
I like this idea! Could you please show us example of such places on OpenStreetMap?
